src/components/common/ProfilePanel.js

```diff
@@ -228,7 +232,7 @@ class ProfilePanel extends React.Component {
     </Link>
   );
```

Comment:
there's a lot of these buttons duplicated. it might be good to have a `renderProfilePanelButton` function and specify with a docstring which buttons can be rendered with this function. It should probably take in `icon`, and `name` or something like that

Feel free to not do this if you don't want to since it's out of scope, but it'd be nice to have